### PR TITLE
Check whether a file is executable before assuming it is runnable

### LINKED ISSUE #638: How to properly use the `man` command?
screen shot 2016-01-21 at 6 43 14 am

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #638.

### CODE REVIEW

File: xonsh/built_ins.py [hunk 1]
```diff
@@ -650,6 +697,7 @@ def subproc_uncaptured(*cmds):
     """Runs a subprocess, without capturing the output. Returns the stdout
     that was produced as a str.
     """
+    print(cmds)
```
Comment (scopatz) [inline xonsh/built_ins.py]: Looks like an errant print statement here.

Comment (blink1073) [inline xonsh/built_ins.py]: Removed.

**Comment by scopatz** [comment]
This looks good to me.  Would someone with access to Windows mind trying it out on their system.  Or did you test this out on Windows @blink1073?

**Comment by blink1073** [comment]
I've only tested on OSX, let me fire up the Windows VM.

**Comment by blink1073** [comment]
No ill affects, `ipconfig` works, and `print('hello')` works.

**Comment by scopatz** [comment]
OK Great!

**Comment by scopatz** [comment]
Thanks a ton @blink1073!

**Comment by blink1073** [comment]
No, thank *you* and @gforsyth.

**Pull request merged**